core/baseline-core-v1.0-psd01.md

```diff
@@ -819,65 +825,100 @@ The identifiers and identity utilized in a BPI MUST be controlled by its Princip
+
 #### **[CR1]>[D11]**	
 Any Principal Owner in a DPKI system utilized by a BPI MUST be able to broadcast a message if it is well-formed within the context of the DPKI.
 
 *Other peers in the system do not require admission control. This implies a decentralized consensus mechanism naturally leading to the utilization of systems such as CCSMs.*
 
+[[CR1]>[D11]](#cr1>d11) Testability: DID methods that have a message protocol conformant to the [W3C DID Standard](https://www.w3.org/TR/did-core/#methods) can be used to broadcast messaged within a DPKI system. A DID method must have the capacity to send Create, Read, Update, and Delete messages. An example of a did method that meets such requirements is ["did:ethr"](https://github.com/decentralized-identity/ethr-did-resolver/blob/master/doc/did-method-spec.md).
+
 #### **[CR2]>[D11]**	
 Given two or more histories of DPKI updates, any Principal Owner within a BPI MUST be able to determine which one is preferred due to security by inspection.
 
 *This implies the existence of a method of ascertaining the level of resources backing a DPKI history such as the hash power in Bitcoin based on difficulty level and nonce.*
 
+[[CR2]>[D11]](#cr2>d11) Testability: Verification to determine that a set of keys associated with a DID in a DPKI is the valid and latest set of keys can be achieved by:
+1. Using a universal resolver to get the latest set of keys associated with the DID used in a BPI
+2. Extracting the block and transaction hash of the latest update on the CCSM used by the DID method from the metadata of the universal resolver.
+3. Verifying if the latest set of keys are in the longest chain using a block explorer such as [Etherscan](https://etherscan.io/).
+
 *Requirements of Identifier registration in DPKI are handled differently from DNS. Although registrars may exist in DPKI, these registrars must adhere to several requirements that ensure that identities belong to the entities they represent. This is achieved the following way:*
 
 #### **[CR3]>[D11]**	
 Private keys utilized in a BPI MUST be generated in a manner that ensures they remain under the Principal Owner’s control. 
 
+[[CR3]>[D11]](#cr3>d11) Testability: To ensure the Principal Owner is solely reponsible for the creation and use the set of public/private keys associated with their DID, a non-custodial wallet must be used to initiate all transactions on behalf of that principal owner within a BPI. An example of implementing this would be to integrate this non-custodial wallet with the BPI APIs that ensure all API payloads/messages are signed, and therefore authorized, solely by the Principal Owner's private key before executing BPI transactions.
```

Comment:
the fact that you use a non-custodial wallet doesn't seem like it proves the keys are under your control, it's just a best practice. If you tweeted your private key, anyone could use it to sign stuff however they like.

I'm not sure you can enforce this through the key generation. There are standards for key custody that might be good references, e.g. https://csrc.nist.gov/Projects/Key-Management/Key-Management-Guidelines, https://cryptoconsortium.org/cryptocurrency-security-standard-documentation/details/ ...

And a nits - you don't need control of your public key, by definition...

```suggestion
[[CR3]>[D11]](#cr3>d11) Testability: To ensure the Principal Owner is solely reponsible for the creation and use of the private key associated with their DID, a non-custodial wallet must be used to initiate all transactions on behalf of that principal owner within a BPI. An example of implementing this would be to integrate this non-custodial wallet with the BPI APIs that ensure all API payloads/messages are signed, and therefore authorized, solely by the Principal Owner's private key before executing BPI transactions.
```